pyqrack/pauli.py

```diff
@@ -10,10 +10,10 @@
 
 class Pauli(IntEnum):
     # Pauli Identity operator. Corresponds to Q# constant "PauliI."
-    PauliI = 0,
+    PauliI = (0,)
     # Pauli X operator. Corresponds to Q# constant "PauliX."
-    PauliX = 1,
+    PauliX = (1,)
     # Pauli Y operator. Corresponds to Q# constant "PauliY."
-    PauliY = 3,
+    PauliY = (3,)
     # Pauli Z operator. Corresponds to Q# constant "PauliZ."
     PauliZ = 2
```

Comment:
This is a very subtle thing about python, if you want to describe a tuple you write `(1, 2, 3)`, suppose you want to write a tuple that has just one element, doing `(1)` will not create a tuple, you will have to do `(1, )`, this has a shorthand notation of `1,`. So those values have been described as tuples from the beginning just that no one had noticed it.

I will change them to numbers than tuples since that seems to be the intended approach :D